packages/graphback-codegen-resolvers/src/formatters/apollo/jsResolverFormatter.ts

```diff
@@ -0,0 +1,37 @@
+const runtimeImportTemplate = `const { validateRuntimeContext } = require("@graphback/runtime");`
+
+export const resolverFileTemplateJS = (name: string, outputResolvers: string[]) => {
+    return `${runtimeImportTemplate}\n\n exports.${name.toLowerCase()}Resolvers = {
+      ${outputResolvers.join(',\n\n\t')}  
+    }`
+}
+
+export const createResolversIndexJS = (resolverNames: string[], exportName: string = 'resolvers'): string => {
+    const imports = resolverNames.map((name: string) => {
+        return `const { ${name}Resolvers } = require('./${name}')`;
+    }).join('\n');
+
+    const importNames = resolverNames.map((name: string) => `${name}Resolvers`);
+
+    return `${imports}
+
+    exports.${exportName} = [${importNames.join(', ')}];`;
+}
+
+export const rootResolversIndexJS = (groups: string[] = ['generated', 'custom']) => {
+    return `${groups.map((name: string) => `const { ${name}Resolvers } = require('./${name}')`).join('\n')}
```

Comment:
This is what was killing me with the previous implementation. 
Too much logic in template. itself that is hard to maintain. If you take look on the other templates I have been doing for loops outside and using concat. NOt that this is bad, but we target simplicity.